### ISSUE TITLE
We manually remove lint and rubocop every time we run `pdk update`

### ISSUE THREAD

**Issue by npwalker** [puppetlabs/puppetlabs-puppet_metrics_collector#10]
We should setup pdk to remove them for us via configuration or just go through and fix the issues that are brought up by those checks.  

cc @jarretlavallee 
cc @tkishel your new function had a bunch of lint warnings 

```
bundle exec rake $CHECK
---> syntax:manifests
---> syntax:templates
---> syntax:hiera:yaml
functions/hosts_with_pe_profile.pp - ERROR: two-space soft tabs not used on line 25
functions/hosts_with_pe_profile.pp - ERROR: two-space soft tabs not used on line 26
functions/hosts_with_pe_profile.pp - ERROR: two-space soft tabs not used on line 27
functions/hosts_with_pe_profile.pp - ERROR: two-space soft tabs not used on line 28
functions/hosts_with_pe_profile.pp - ERROR: two-space soft tabs not used on line 29
functions/hosts_with_pe_profile.pp - ERROR: two-space soft tabs not used on line 30
manifests/pe_metric.pp - WARNING: selector inside resource block on line 18
manifests/puppetserver.pp - WARNING: indentation of => is not properly aligned (expected in column 24, but found it in column 19) on line 41
manifests/puppetserver.pp - WARNING: indentation of => is not properly aligned (expected in column 24, but found it in column 19) on line 42
manifests/puppetserver.pp - WARNING: indentation of => is not properly aligned (expected in column 24, but found it in column 19) on line 43
manifests/puppetserver.pp - ERROR: two-space soft tabs not used on line 37
```

**Comment by tkishel** [comment]
😿

**Comment by tkishel** [comment]
https://github.com/puppetlabs/puppetlabs-puppet_metrics_collector/pull/11

**Comment by jarretlavallee** [comment]
After #11 is merged, #13 cleans up the rest of the lint issues and reenables the rubocop and lint checks.

**Comment by jarretlavallee** [comment]
Resolved with #20
